drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/widgets/ScenarioGridColumn.java

```diff
@@ -31,14 +32,18 @@
 
     private final ScenarioCellTextBoxSingletonDOMElementFactory factory;
 
+    final ScenarioHeaderMetaData informationHeaderMetaData;
+
     public ScenarioGridColumn(HeaderMetaData headerMetaData, GridColumnRenderer<String> columnRenderer, double width, boolean isMovable, ScenarioCellTextBoxSingletonDOMElementFactory factory) {
         super(headerMetaData, columnRenderer, width);
+        this.informationHeaderMetaData = (ScenarioHeaderMetaData) headerMetaData;
         this.setMovable(isMovable);
         this.factory = factory;
     }
 
     public ScenarioGridColumn(List<HeaderMetaData> headerMetaData, GridColumnRenderer<String> columnRenderer, double width, boolean isMovable, ScenarioCellTextBoxSingletonDOMElementFactory factory) {
         super(headerMetaData, columnRenderer, width);
+        this.informationHeaderMetaData = (ScenarioHeaderMetaData) headerMetaData.stream().filter(hdm -> ((ScenarioHeaderMetaData) hdm).isInformationHeader()).findFirst().orElse(headerMetaData.get(0));
```

Comment:
Are we sure `List<HeaderMetaData> headerMetaData` contains always at least one element? Asking because of `.orElse(headerMetaData.get(0))`

Reply:
yes, our columns always has at least one header. I agree it is a strong assumption, but it is how they works.

Reply:
ok